### PR TITLE
Slushie milestone 1 delivery report

### PR DESCRIPTION
**Pull request by Kyrylog**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/1004 < please fill this in with the PR number of your application.

### CODE REVIEW

File: deliveries/Slushie-Milestone_1.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Milestone Delivery :mailbox:
+> :zap: Only the GitHub account that submitted the application is allowed to submit milestones.
+>
+> Don’t remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
+* **Application Document:** Please, provide a link to the merged contract: [link](https://github.com/w3f/Grants-Program/blob/master/applications/Slushie%20Mixer.md)
```
Comment (alxs) [inline deliveries/Slushie-Milestone_1.md:6]: Not sure it works with the space, but it's what's causing the action to fail

```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/Slushie Mixer.md
```

Comment (alxs) [reply deliveries/Slushie-Milestone_1.md]: I fixed the action to also work with spaces

**Comment by Kyrylog** [comment]
Thank you

вт, 19 лип. 2022 р. о 20:21 Aleixo Sanchez ***@***.***> пише:

**Comment by CrommVardek** [comment]
Hi @Kyrylog , I've an in progress evaluation for your delivery. 

Everything is good to me except the documentation step. I was expecting to see the "details used libraries & contract structure." as specified in the application's deliverable specification.

Once this is added to the readme file, I'll change my evaluation to accepted and make a PR for it. 

https://github.com/CrommVardek/Grant-Milestone-Delivery/blob/slushie-mixer-eval-cromm-vardek/evaluations/slushie_mixer_1_cromm_vardek.md

**Comment by Kyrylog** [comment]
Hi Cyril,

We’ve added file
https://github.com/4IRE-Labs/Slushie/blob/main/usage.md



чт, 21 лип. 2022 р. о 21:58 Cyril Carlier ***@***.***> пише:

**Comment by CrommVardek** [comment]
It's good to me, I've adapted my evaluation accordingly: https://github.com/CrommVardek/Grant-Milestone-Delivery/blob/slushie-mixer-eval-cromm-vardek/evaluations/slushie_mixer_1_cromm_vardek.md

**Comment by semuelle** [comment]
Hi @Kyrylog. I am happy to say that I have nothing to add to @CrommVardek's evaluation and your milestone is hereby accepted. The evaluation notes are now merged [here](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/slushie_mixer_1_cromm_vardek.md). I'm looking forward to your next deliveries!

I will forward your invoice for processing.

**Comment by RouvenP** [comment]
hi @Kyrylog we transferred the payment today. Thanks!

**Pull request merged**